components/Collections/examples/CollectionsSectionInsetsExample.m

```diff
@@ -0,0 +1,114 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+#import "CollectionsSectionInsetsExample.h"
+
+static const NSInteger kSectionCount = 5;
+static const NSInteger kSectionItemCount = 3;
+static NSString *const kReusableIdentifierItem = @"itemCellIdentifier";
+
+@implementation CollectionsSectionInsetsExample {
+  NSMutableArray *_content;
```

Comment:
Usually we use @propertys in an objc extension and let the ivars be free. If you're overriding both the setter and the getter you'll _also_ have to declare an ivar but that's a super exception.